spec/unit/driverProviders/direct_test.js

```diff
@@ -0,0 +1,69 @@
+
+var path = require('path');
+var fs = require('fs');
+var BrowserError = require('../../../built/exitCodes').BrowserError;
+
+var removeSeleniumFolder = function() {
```

Comment:
Instead of doing this, just use a config file that points at a different (and invalid) chromedriver location.